tests/installation/releasenotes.pm

```diff
@@ -27,7 +27,12 @@ sub run() {
         send_key "alt-e";    # open release notes window
     }
     else {
-        send_key "alt-l";    # open release notes window
+        if (check_var('VIDEOMODE', 'text')) {
+            send_key "alt-l";    # open release notes window
+        }
+        else {
+            assert_and_click('release-notes-button');
+        }
```

Comment:
alt-l works in SLE 12 SP2 and SP1...this poor gentleman has to test SLE 12 GA...